javase/src/main/java/com/google/zxing/client/j2se/DecodeWorker.java

```diff
@@ -165,6 +165,17 @@ private static void dumpResult(URI input, Result... results) throws IOException
             result.getText() + "\n" +
             "Parsed result:\n" +
             parsedResult.getDisplayResult() + "\n");
+
+        if (config.outputRaw) {
+            String rawData = "";
```

Comment:
Total nits but can you switch to 2-space indent? and this could be a StringBuilder

Reply:
Sure (strange that I missed the indent, I blame not bothering to set the correct offset in my editor for such a small change). I also switch to StringBuilder, it makes sense even though it isn't critical here. (I didn't look hard enough for Python's string.join())